extensions/adobe/experience/adcloud/search/adgroup-all.example.1.json

```diff
@@ -0,0 +1,16 @@
+{
+    "@id": 1792062465,
```

Comment:
`@id` is special in JSON-LD.  You probably want to be using `xdm:id`

Reply:
Replaced example with xdm:id